src/common/low_precision_transformations/tests/add_transformation.cpp

```diff
@@ -439,6 +443,23 @@ const std::vector<AddTransformationTestValues> testValuesWithoutConstantBranches
       {{}, {}, {10.f}},
       {}},
      "convolution"},
+    // convolution before FQ
+    {false,
+     -1,
+     LayerTransformation::createParamsU8I8(),
+     {ngraph::element::u8,
+      {{ngraph::element::f32}, {7.f}, {10.f}},
+      ngraph::element::u8,
+      {{ngraph::element::f32}, {3.f}, {5.f}},
+      {}},
+     {ngraph::element::u8,
+      {{}, {}, {}},
+      ngraph::element::u8,
+      {{ngraph::element::f32}, {17.f}, {0.5f}},
+      {{}, {}, {10.f}},
+      {}},
+     "convolution",
+     "bias_on_zero_input"},
```

Comment:
I'm just interesting: is clang formatting more understandable for you : )))

Reply:
In test instances definitely no) But I wanted to avoid additional changes in this PR (I mean changing formatting for all test instances in this file)